src/transactions/feed/TransactionFeedV2.test.tsx

```diff
@@ -139,30 +139,26 @@ describe('TransactionFeedV2', () => {
     await waitFor(() => expect(tree.getAllByTestId('TransferFeedItem').length).toBe(2))
   })
 
-  it("doesn't render transfers for tokens that we don't know about", async () => {
-    mockFetch.mockResponse(typedResponse({ transactions: [mockTransaction()] }))
```

Comment:
i deleted this test because `mockTransaction()` actually is a token that we _do_ know about so i don't think this test is doing what it says. also, we should rely on the api to return supported tokens